doc/index.rst

```diff
@@ -80,7 +80,7 @@ Usage
    meta.ipynb
    fairness.ipynb
    outliers.ipynb
-   timegapsplit.ipynb
+   crossvalidation.ipynb
```

Comment:
While rendering I found this mismatch and move forward to change it in the index.